api/api/tasks/chat_task_schema_generation/schema_generation_agent.py

```diff
@@ -108,4 +103,17 @@ async def run_agent_schema_generation(
     - 'document_file' allows to support indistinctively text (txt, json, csv, etc), images and pdfs file.
     - If 'available_tools_description' is provided, consider how these tools might be utilized in the agent and adjust the schema accordingly.
     - For cases where the agent requires static or infrequently updated context that does not vary from agent run to agent run, you do not need to include this context in the input schema. Instead, explain in the 'answer_to_user' that the agent instructions are the best place this context. Task instructions are outside of your scope and are generated afterwards by another agent, do not offer to update the instructions. Non-exhaustive examples of large and static content: FAQ knowledge base for customer service agents, Company policies or guidelines for compliance checking agents, Style guides for content creation agents, Standard operating procedures for process analysis agents, reference documentation for technical support agents, etc. As a rule of thumbs, input data that  is supposed to change every time the agent is run can go in the 'input_json_schema', input data that varies way rarely can go in the instructions."""
-    ...
+
+
+# TODO: switch back to Claude 3.7 when we'll have more token quotas
```

Comment:
You should switch now ! I'll add the quotas in https://github.com/WorkflowAI/WorkflowAI/pull/21